test/tests/reporters/constant_reporter/constant_reporter.i

```diff
@@ -0,0 +1,39 @@
+[Mesh/mesh]
+  type = GeneratedMeshGenerator
+  dim = 1
+[]
+
+[Problem]
+  kernel_coverage_check = false
+  solve = false
+[]
+
+[Reporters]
+  active = constant
+  [constant]
+    type = ConstantReporter
+    names          = 'int_1   str    int_2   num_1 num_2 vec_1  int_3   vec_2'
+    value_types    = 'integer string integer real  real  vector integer vector'
```

Comment:
I am thinking that you have: int_names, int_values, real_names, real_values, vector_int_names, vector_int_values, etc. This way it always be a one-to-one on the number of inputs and the value_types parameter goes away.

Reply:
Good idea. This will clean up my logic in the constructor a lot. I keep wanting to make it like python for some reason.